integration_tests/tsconfig.json

```diff
@@ -0,0 +1,42 @@
+{
+  "compilerOptions": {
+    "outDir": "testlib",
+    "rootDirs": ["stacks", "../src"],
+    "alwaysStrict": true,
+    "charset": "utf8",
+    "declaration": true,
+    "experimentalDecorators": true,
+    "incremental": true,
+    "inlineSourceMap": true,
+    "inlineSources": true,
+    "lib": [
+      "es2018"
+    ],
+    "module": "CommonJS",
+    "moduleResolution": "node",
+    "newLine": "lf",
+    "noEmitOnError": true,
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "resolveJsonModule": true,
+    "strict": true,
+    "strictNullChecks": true,
+    "strictPropertyInitialization": true,
+    "stripInternal": false,
+    "target": "ES2018",
+    "composite": false,
+    "tsBuildInfoFile": "testlib/tsconfig.tsbuildinfo"
+  },
+  "include": [
+    "stacks/**/*.ts",
+    "../version.json"
+  ],
+  "exclude": [
+    "../node_modules"
+  ]
+}
+  
```

Comment:
Do we need a separate tsconfig.json just for the integration tests?

Reply:
(Maybe we do, just curious)

Reply:
I believe so -- our integration test compiles some test typescript files and rather than passing in flags to the `tsc` command, this was the easier option.